examples/redis-cache-failure-handling/build.gradle

```diff
@@ -5,6 +5,10 @@ buildscript {
 	repositories {
 	  maven {
 	    url System.getenv('CICD_ARTIFACTORY_URL') ?: ''
+	    credentials {
```

Comment:
everything looks fine, except cosmetic changes of indentation.

Reply:
Done